dali/operators/util/randomizer.cuh

```diff
@@ -130,6 +130,19 @@ struct curand_uniform_int_values_dist {
   int64_t nvalues_ = 0;
 };
 
+struct curand_bernoulli_dist {
+ public:
+  explicit DALI_HOST_DEV curand_bernoulli_dist(float probability = 0.5f)
+    : probability_(probability) {}
+
+  __device__ inline bool operator()(curandState *state) const {
+    return curand_uniform(state) <= probability_ ? true : false;
```

Comment:
How about
```suggestion
    return !(curand_uniform(state) > probability_);
```
;) Up to you

Reply:
I could do `return curand_uniform(state) <= probability_`.
The negation just complicates things I believe

Reply:
Yeah, that's true ;)